app/components/root.js

```diff
@@ -1,22 +1,38 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import {
   BrowserRouter as Router,
   Switch,
   Route,
   Redirect
 } from 'react-router-dom';
+import { fetchUser } from '../redux/thunks/userThunks';
 import CreateRoute from './CreateRoute';
 import Home from './Home';
 import Navigation from './Navigation';
+import Login from './Login';
 
 class Root extends Component {
+  componentDidMount() {
+    const { fetchUser } = this.props;
+    let fetchUserResult = fetchUser(
+      document.cookie
+        .split(';')
+        .filter(c => /session_id=/.test(c))[0]
+        .replace(/session_id=/, '')
+        .replace(/ /, '')
```

Comment:
Getting an error of _"Cannot read property 'replace' of undefined"_ in the console and unable to load the page, not sure if we need this part of the componentDidMount because it looks we're fetching a user by their sessionId by the global variable cookie

Reply:
We don't need fetchUser in componentDidMount. It's been removed.